code/modules/paperwork/photocopier.dm

```diff
@@ -56,6 +60,8 @@
 		return
 	if(href_list["copy"])
 		if(copy)
+			if(!SEND_SIGNAL(src, COMSIG_OBJ_ATTEMPT_CHARGE_PRINTER, usr))

```

Comment:
did you intend on only having copies and not photocopies and doc copies cost money, as they are free below in the else ifs